modules/developers-guide/pages/webpack-config.adoc

```diff
@@ -99,6 +154,93 @@ const Navigation = () => {
 export default Navigation;
 ----
 
+== Iterate faster using webpack-dev-server
+
+There are a few simple shortcuts you can use to iterate faster in your development environment. 
+For example, you can choose to build and deploy only specific canisters instead of all canisters in a project.
+
+If most of your changes are in the front-end for your application, one of the most effective ways you can iterate faster is by installing and configuring the webpack development server.
+THe webpack development server—`+webpack-dev-server+`—provides in-memory access to the webpack assets, enabling you to make changes and see them reflected in the browser right away using live reloading.
+
+To install and configure `+webpack-dev-server+`:
+
+. Create a new project and change to your project directory.
+. Install the `+webpack-dev-server+` in the project directory by running the following command:
++
+[source,bash]
+----
+npm install webpack-dev-server
+----
+. Open the `+webpack.config.js+` file for your project in a text editor.
+. Add your network host name and port information after the `+output+` section of the `+webpack.config.js+` file.
++
+For example, if you are using the default host and port information for local development, you would add the following to the `+webpack.config.js+` file:
++
+[source,bash]
+---- 
+devServer: {
+ proxy: {
+   "/api": "http://localhost:8000",
+ },
+},
+----
```

Comment:
Not entirely confident this is necessary, but it very well could be. Worth double-checking with @krpeacock, but I did not need this in my create-react-app config (separate but similar project)